README.md

```diff
@@ -412,6 +412,41 @@ markdown link isn't linked).
 It's best to only inject HTML that is self-contained and doesn't depend upon
 other elements on the page, since those may change in future versions of Dartdoc.
 
+### Skipping constant rendering with one-line docs
+
+For some classes or libraries full of well-documented constants, showing the
+implementation on the enclosing `class` or `library` page can be distracting
+or even misleading.  To prevent the rendering of constant implementations,
+place the `{@hideConstantImplementations}` in the documentation comment for
+the enclosing context where the constant is defined.   For members of a class,
+place the directive in the class documentation where the constants are defined.
+For top level constants, place the directive in the library where the constants
+are defined.
+
+For example:
+
+```dart
+/// This is truly an amazing library.
+/// {@hideConstantImplementations}
+library my_library;
```

Comment:
nit: I think we're steering away from named libraries like this; we can use `library;` in Dart 2.19.